### ISSUE TITLE
Links Have a strange flicker when shift+tabbing before them

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#737]
### Expected Behaviour
Links should show a clear focus indicator when visited and do nothing when focus is changed elsewhere in the page

### Actual Behaviour
Links sometimes don't show focus and sometimes flicker when changing focus elsewhere on the page

#### Steps to Reproduce

1. Run the sample page and tab to the first link (standard link)
1. Note that no focus indicator is shown
1. Tab past the 3rd link to the slider
2. Shift+Tab to go back to the links
2. Note that double underlines are shown on each of the links 
2. Shift+tab until focus is before the links
2. Note the flickering that occurs

#### Platform and Version
Chrome 83 / main branch
Links do not flicker with Safari Technology Preview (but the focus issue is still there)

#### Sample Code that illustrates the problem
https://github.com/jnurthen/spectrum-web-components/tree/linkFlickerIssue/projects/example-project/src/index.html
https://github.com/jnurthen/spectrum-web-components/tree/linkFlickerIssue/projects/example-project/src/index.js

**Comment by jnurthen** [comment]
video showing issue at https://raw.githack.com/jnurthen/spectrum-web-components/linkFlickerIssue/projects/example-project/src/Flicker.mov

**Comment by Westbrook** [comment]
When leveraging `:focus-visible` natively, I am no longer experience this issue.

Even in Safari (where `:focus-visible` is not available) I am no longer experiencing this when leveraging the following experiences:
![image](https://user-images.githubusercontent.com/1156657/117459989-01d9e600-af1a-11eb-910e-607b6e79fb4e.png)

This is using both the CSS and the `sp-theme` based theme application.

If a version of this can be recreated at some point, it'll probably benefit from a new issue, anyways, so I'll be closing this one.
